scripts/requirements-base.txt

```diff
@@ -12,6 +12,7 @@ PyYAML>=5.1
 
 # YAML validation. Used by zephyr_module.
 pykwalify
+strictyaml
```

Comment:
Probably you are right, but I didn't find any clear explanation how this requirements files are divided - I thought that this is proper place for this package, because there are also placed other packages used for `yaml` parsing.